pkg/cmd/autocomplete.go

```diff
@@ -53,9 +63,19 @@ func NewAutocompleteBash(g autocomplete.Generator) *cobra.Command {
 
 	return &cobra.Command{
 		Use:     bash.String(),
-		Short:   "Add bash autocomplete for terminal",
-		Long:    "Add bash autocomplete for terminal",
-		Example: "rit completion bash",
+		Short:   "Add bash autocomplete for terminal, --help to know how to use",
+		Long:    `
+Add bash autocomplete for terminal
+Only works if bash auto completion is installed.
+
+To test run: 
+ $ rit completion bash | source
+
+To install run: 
+ $ echo "[[ -r "/usr/local/bin/rit" ]] && rit completion bash > ~/.rit_completion" >> ~/.bashrc
```

Comment:
```suggestion
 $ echo "[[ -r "/usr/local/bin/rit" ]] && rit completion bash | source" >> ~/.bashrc
```
i guess only save completion on ~/.rit_completion don't work if the user not call "source ~/.rit_completion  "

Reply:
I added the line:

` echo "source ~/.rit_completion" >> ~/.bashrc`
To solve the missing stuff.
The reason we need to do this way is that when running on macOS stuff breaks if you do like you mentioned. I don't know why LOL.